src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/RequirementTypeMainCategory.java

```diff
@@ -33,4 +33,8 @@
 
     @Column(name="RESTRICTIVE")
     private String restrictive;
+
+    public boolean isRarRequirementType() {
+        return code != null && code.equals(REHABILITATION_ACTIVITY_REQUIREMENT_CODE);
+    }
```

Comment:
very minor style thing I guess you could have written `return REHABILITATION_ACTIVITY_REQUIREMENT_CODE.equals(code);`